### ISSUE TITLE
Bot Doesn't Recognize Setting Changes Until Reconnected

### ISSUE THREAD

**Issue by clarkio** [clarkio/vscode-twitch-highlighter#102]
**Describe the bug**
If a user starts the extension to connect to chat and then changes settings those updates will not be reflected until the extension is disconnected and reconnected. For example if you change the setting for response to and empty command, `!line`, then it will not be changed until reconnected.

**Comment by parithon** [comment]
I agree with this. A redesign of how settings are conveyed to the server is needed. Perhaps this can be accomplished during a major refactoring process prior to version 1.0?
